pyocd/target/builtin/__init__.py

```diff
@@ -216,6 +217,7 @@
           'stm32l031x6' : target_STM32L031x6.STM32L031x6,
           'stm32h723xx' : target_STM32H723xx.STM32H723xx,
           'stm32h743xx' : target_STM32H743xx.STM32H743xx,
+          'stm32h7b0' : target_STM32H7B0.STM32H7B0,
```

Comment:
Would you mind adding xx's to the target name so it becomes `stm32h7b0xx`? (Granted the F051 is not like that, but that's historical…)